README.md

```diff
@@ -104,7 +104,8 @@ For more information, see the [Fastify CLI documentation](https://github.com/fas
 
 #### Note
```

Comment:
Since we are making a change here, let's replicate the call to attention format that is used in the getting started guide:

```md
> ## Note
> (the text of the note)
```
Yields:
> ## Note
> (the text of the note)